cola/themes.py

```diff
@@ -521,9 +525,29 @@ def style_sheet_flat(self):
             focus=focus,
         )
 
+    def style_sheet_custom(self, app_palette):
+        """Get custom style sheet.
+        File name is saved in variable self.name.
+        If user has deleted file, use default style"""
+
+        # get user path Windows
+        userpath = os.getenv('USERPROFILE')
+        # if not found Windows, try Unix
+        if not userpath:
+            userpath = os.getenv('HOME')
+        # check if path exists
+        filepath = os.path.join(
+            userpath, '.config', 'git-cola', 'themes', self.name + '.qss'
+        )
```

Comment:
There's some duplicate code here. I think `cola/resources.py` is where a helper function `def custom_theme_path(name)` could go.

I *think* this section can be replaced with:

```
filepath = resources.config_home('themes', self.name + '.qss')
```

That way it respects the XDG .config variables, etc. We don't really handle USERPROFILE in there, though, but I want to believe that the `expanduser('~')` in Python is doing the right thing and handling that for us.

If it's not, then that would be a good place to centralize that behavior and have an `if utils.is_win32(): ...` special-case where we check for USERPROFILE.

Reply:
it is resources.config_home, I didn't know about this, learned something today!